vision/getting-started/image_editing.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    SEGMENTATION_CLASS = 16  # @param {type: 'number'}
</pre><p>This is using a different type than before. I'd suggest using one or the other, if possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Updated</p>